Pathing issues on pypi upload on Windows

BUILD FAILED - [Errno 22] invalid mode ('w') or filename: 'c:\\Users\\jcorson\\pythonWorkspace\\projectNameHere\\target/reports\\distutils\\sdist__upload__-r__http:123.123.120.117:8080'

I am on windows working in the Git Bash. pybuilder version 0.11.8. I saw another closed issue about pathing and you made him upgrade but this is very recent

This issue didn't exist when it was the public PyPI server but I changed it to a local pypi that I need to store my project in. As a result, it seems to append the -r and the URL I provided

project.set_property("distutils_upload_repository", 'http://123.123.120.117:8080')

@arcivanov Well i tried

```
$ pip install --force-reinstall git+git://github.com/arcivanov/pybuilder@issue_387
Collecting git+git://github.com/arcivanov/pybuilder@issue_387
  Cloning git://github.com/arcivanov/pybuilder (to issue_387) to c:\users\jcorson\appdata\local\temp\pip-xoxpop-build
    Complete output from command python setup.py egg_info:
    PyBuilder version ${dist_version}
    Build started at 2016-08-26 11:27:37
    ------------------------------------------------------------
    [INFO]  Building PyBuilder version 0.11.10.dev (0.11.10.dev20160826152737)
    [INFO]  Executing build in c:\users\jcorson\appdata\local\temp\pip-xoxpop-build
    [INFO]  Going to execute tasks: clean, install_dependencies, package
    [INFO]  Removing target directory c:\users\jcorson\appdata\local\temp\pip-xoxpop-build\target
    [INFO]  Installing all dependencies
    [INFO]  Processing batch dependency 'pip>=7.1'
    [INFO]  Installing plugin dependency pdoc
    [INFO]  Installing plugin dependency sphinx
    ------------------------------------------------------------
    BUILD FAILED - Missing prerequisite sphinx required by plugin python.sphinx
    ------------------------------------------------------------
    Build finished at 2016-08-26 11:28:05
    Build took 28 seconds (28137 ms)

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in c:\users\jcorson\appdata\local\temp\pip-xoxpop-build\
```

--- a/src/main/python/pybuilder/plugins/python/distutils_plugin.py
+++ b/src/main/python/pybuilder/plugins/python/distutils_plugin.py
@@ -35,7 +35,7 @@ from pybuilder.core import (after,
                             RequirementsFile,
                             Dependency)
 from pybuilder.errors import BuildFailedException
-from pybuilder.utils import as_list, is_string, is_notstr_iterable, get_dist_version_string
+from pybuilder.utils import as_list, is_string, is_notstr_iterable, get_dist_version_string, safe_log_file_name
 from pybuilder.pip_utils import build_dependency_version_string
 from textwrap import dedent
 from pybuilder.pip_utils import pip_install
@@ -257,9 +257,9 @@ def execute_distutils(project, logger, distutils_commands, clean=False):
     for command in distutils_commands:
         logger.debug("Executing distutils command %s", command)
         if is_string(command):
-            output_file_path = os.path.join(reports_dir, command.replace("/", ""))
+            output_file_path = os.path.join(reports_dir, safe_log_file_name(command))
         else:
-            output_file_path = os.path.join(reports_dir, "__".join(command).replace("/", ""))
+            output_file_path = os.path.join(reports_dir, safe_log_file_name("__".join(command)))
         with open(output_file_path, "w") as output_file:
             commands = [sys.executable, setup_script]
             if project.get_property("verbose"):

